src/problems_truss.jl

```diff
@@ -0,0 +1,65 @@
+# This file is a part of JuliaFEM.
+# License is MIT: see https://github.com/JuliaFEM/JuliaFEM.jl/blob/master/LICENSE.md
+
+# This is copied from the files problem_elasticity.jl
+"""
+This truss fromulation is from
+"""
+type Truss <: FieldProblem
+    # these are found from problem.properties for type Problem{Elasticity}
+    formulation :: Symbol
+    # Maybe we need more here
+end
+function Truss()
+    # formulations: plane_stress, plane_strain, continuum
+    return Truss(:plane_stress)
```

Comment:
If you remove that formulation definition from the type, here we just `return Truss()`. Actually, this initializator can be in that case removed because it's not doing anything else than giving infinite loop. The use of initializator is needed only when we have internal properties for Problem to have some good default.